src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -140,56 +126,120 @@ public void setDescription(String description) {
   }
 
   /**
-   * Adds an admin user to project
+   * Add a regular user to the project
+   *
+   * @param userId id of the user to add
+   */
+  public void addRegularUser(long userId) {
+    addUser(userId, false);
+  }
+
+  /**
+   * Add an admin user to the project
    *
    * @param userId id of the user to add
    */
   public void addAdminUser(long userId) {
-    // remove user if they're already in project
-    if (this.userIds.keySet().contains(userId)) {
-      removeUser(userId);
-    }
-    this.userIds.put(userId, User.ADMIN);
+    addUser(userId, true);
   }
 
   /**
-   * Adds a regular user to the project
+   * Helper function for addRegularUser() and addAdminUser() to add any user to the project. Can
+   * only add regular/admin users
    *
    * @param userId id of the user to add
+   * @param isAdmin is the user an admin?
    */
-  public void addRegularUser(long userId) {
+  private void addUser(long userId, boolean isAdmin) {
```

Comment:
Mayber pass the userrole enum instead of a boolean since you have 3 different types.  I understand the 3rd type being the `CREATOR` it wouldn't change and wouldn't need to be accounted for by addUser.  But in the future when you add a 4th type (maybe VIEWER or AUDITOR, etc..) you'd then have to either add more bools here or change this into an enum anyways.